libs/zoekt/src/lib/zoekt.module.ts

```diff
@@ -0,0 +1,29 @@
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { NgModule } from '@angular/core';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { MatCardModule } from '@angular/material/card';
+import {
+  Route,
+  RouterModule,
+  UrlMatchResult,
+  UrlSegment
+} from '@angular/router';
+
+import { ZoektComponent } from './zoekt.component';
+import { ZoektService } from './zoekt.service';
+
+export const routes: Route[] = [{ path: 'search', component: ZoektComponent }];
```

Comment:
The importing module already search that /search should match this module. I believe that this path should be ''